packages/bvaughn-architecture-demo/components/console/renderers/shared.module.css

```diff
@@ -135,18 +135,41 @@
   color: var(--color-error);
 }
 
+.BadgeContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 0.5rem;
+  height: 0.5rem;
+  position: absolute;
+  top: 0.5rem;
+  left: 0.5rem;
+}
+
 .ColorBadge,
 .UnicornBadge {
-  position: relative;
-  display: inline-block;
   width: 0.5rem;
   height: 0.5rem;
-  margin: 0 0.5rem 0 -1rem;
+  box-shadow: 0 0 0 rgba(185, 185, 185, 0.4);
```

Comment:
Any colors (including shadows) should be a CSS variable [defined here](https://github.com/replayio/devtools/blob/main/packages/bvaughn-architecture-demo/pages/variables.css) so we can easily tweak it for each theme (and so we don't end up with a lot of near-neighbor values spread around).